tools/delay_detector/src/lib.rs

```diff
@@ -75,3 +83,13 @@ impl<'a> Drop for DelayDetector<'a> {
         }
     }
 }
```

Comment:
To avoid a lot of cfgs here, I'd do something like this: 

```
#[cfg(feature = "delay_detector")]
mod enabled;
#[cfg(feature = "delay_detector")]
pub use enabled::DelayDetector;

#[cfg(not(feature = "delay_detector"))]
mod disabled;
pub use disabled::DelayDetector;
```

Reply:
Fixed.